src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Consider adding a fallback font-family in case 'Roboto' isn't available. For example, 'font-family: Roboto, Arial, sans-serif;' ensures that if 'Roboto' doesn't load, the browser will try 'Arial' next, and then the generic 'sans-serif' font.